I wish_If only/sofia_zahoruiko.md

```diff
@@ -0,0 +1,29 @@
+1. Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+if only you had come earlier to assess all inferiority of architecture proposed by a first-year student
+
+2. Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+if only we waited for his arrival, we would keep abreast on what stage is the project now and what we should do with it next
+
+3. Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+i wish we had met before leaving and discuss the reason of your dismissal, i have a lot of thoughts in this case and i want to share it with you
+
+4. Жаль что проект завершился так и не начавшись ввиду нехватки кадров и соответствующего финансирования. Если бы все сложилось то задумка вполне могла выстрелить на рынке и стать новым хитом.
+i wish project had more people and appropriate funding to not ends as soon as it starts. If everything gone well, then the idea could well shoot in the market and become a new hit.
+
+5. Он пожалел, что пришел и привел с собой друга. На его месте я бы так не поступал. Если бы он пришел вовремя, этого могло и не случиться.
+He regretted he have come and take a friend with him. in his stead i didnt do this. if only he had come in time it might not have happened.
+
+6. Если бы не ты я не знаю что бы я тогда ответил заказчику, ты повел разговор совсем в другом направлении и нам удалось избежать серьезного разговора о горящих сроках.
+if there weren't you i would don't know what to say to the customer, you took the conversation in a different way and we managed to avoid a serious conversation about deadlines.
+
+7. Жаль что я не могу прочесть эту книгу в оригинале, иначе бы я мог узнавать он новых тенденциях в мире разработки из первоисточника не ожидая годы пока её переведут.
+i wish i could read this book in the original, then i could get to know about new trends in the development world from the source, without waiting for years for translation
+
+8. Жаль что у нас мало времени, иначе бы я смогла заложить шикарную архитектуру и реализовать проект на твердую пятерочку. А опыт у вас в этом есть? Нет, но я читала как это можно сделать.
+i wish we could have more time for making a great architecture and realize the project on a solid five. Do you have experience in this? i dont, but i read the way how to do it.
+
```

Comment:
`If only we had more time so I could set up an amazing architecture` - but your option is also ok
PLEASE DON'T USE `REALIZE` use Implement instead
`for` a solid five